react-components/table-intro/step2.md

```diff
@@ -49,3 +49,5 @@ cells={columns} rows={defaultRows}
 The table should now look like the image below:
 
 <img src="table-intro/assets/step-2-complete.png" alt="Image of what table looks like at the end of step 2." style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+
+Still have questions? View the latest <a href="https://www.patternfly.org/v4/documentation/react/components/table/" target="_blank">Table documentation</a> for more information.
```

Comment:
Should this be []() syntax for links in markdown?

Reply:
Oh geez, good catch! Updated to markdown syntax